app/addons/setup/__tests__/singleNode.test.js

```diff
@@ -0,0 +1,50 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+import utils from "../../../../test/mocha/testUtils";
+import React from "react";
+import ReactDOM from "react-dom";
+import Store from '../../../core/store';
+import {mount} from 'enzyme';
+import SingleNodeController from "../components/SingleNodeController";
+
+const assert = utils.assert;
+
+//this was commented out. I imagine it needs to be updated
```

Comment:
I did mostly manual testing since it's would require to mock all the API layer. I can update those tests but I will only be able to do unit tests on components

Reply:
No its fine. Just delete the tests that are set to skip

Reply:
I'm kind of busy for the rest of the day. I'll do that for tomorrow